Makefile

```diff
@@ -192,6 +192,13 @@ compile-documentation-scripts:
 compile-samples:
 	${ROOTDIR}/node_modules/.bin/tsc --project ./documentation/samples/tsconfig.json
 
+.PHONY: validate-samples
+validate-samples:
+	find documentation/samples/packs -name "*.ts" | \
+	parallel -k \
+	'echo "Validating {}"; node -r ts-node/register --no-deprecation cli/coda.ts validate {}'
```

Comment:
Also running cli/coda.ts I think is kinda slow because it has to compile everything, dunno if it's possible to run the pre-built version in `dist/` for this.

Reply:
Wow, much faster. Thanks!